src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -382,7 +391,8 @@ def _run(args: argparse.Namespace, methods: Sequence[str],
             qps = 0
         channel_config = _ChannelConfiguration(
             method, per_method_metadata.get(method, []), qps, args.server,
-            args.rpc_timeout_sec, args.print_response)
+            args.rpc_timeout_sec, bool(args.print_response),
```

Comment:
`bool("False") == True`?

Reply:
You're right. Meant to use `ast.literal_eval` here. Good catch!

Reply:
Added a more robust boolean argument parser. Will now correctly match "true", "false", "yes", "no", "y", "n", and all variations on capitalization.